liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojoSupport.java

```diff
@@ -23,30 +23,71 @@
 import org.apache.commons.lang3.Validate;
 import org.apache.maven.artifact.Artifact;
 import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.project.MavenProject;
 import org.apache.tools.ant.taskdefs.Copy;
+import org.codehaus.mojo.pluginsupport.util.ArtifactItem;
 import org.w3c.dom.Element;
 
+import io.openliberty.tools.ant.DeployTask;
 import io.openliberty.tools.ant.SpringBootUtilTask;
 import io.openliberty.tools.maven.server.PluginConfigSupport;
 import io.openliberty.tools.maven.utils.MavenProjectUtil;
 import io.openliberty.tools.common.plugins.config.ApplicationXmlDocument;
 import io.openliberty.tools.common.plugins.config.LooseApplication;
 import io.openliberty.tools.common.plugins.config.LooseConfigData;
 
+import io.openliberty.tools.ant.DeployTask;
+import io.openliberty.tools.ant.SpringBootUtilTask;
+import io.openliberty.tools.common.plugins.config.ApplicationXmlDocument;
+import io.openliberty.tools.common.plugins.config.LooseApplication;
+import io.openliberty.tools.common.plugins.config.LooseConfigData;
+
 /**
- * Install artifact into Liberty server support.
+ * Support for installing and deploying applications to a Liberty server.
  */
-public class InstallAppMojoSupport extends PluginConfigSupport {
+public class DeployMojoSupport extends PluginConfigSupport {
+    /**
+     * A file which points to a specific module's war | ear | eba | zip archive location
+     */
+    protected File appArchive;
```

Comment:
I'm a little confused why appArchive and appArtifact are still here, since you removed them as parameters.

Reply:
Using them for the DeployTask.